stl/inc/xtzdb.h

```diff
@@ -29,8 +29,38 @@ struct __std_tzdb_registry_leap_info {
     uint16_t _Reserved;

```

Comment:
Can we rename this header to `__msvc_tzdb.hpp` to be consistent with Billy's proposed naming structure for internal headers? (Which we really should pull the trigger on universally.)

Reply:
I will file this as a card because I don't trust git to handle the rebasing of my PRs in waiting,